nodes/containers/nodes-containers-using.adoc

```diff
@@ -25,6 +25,22 @@ for years. {product-title} and
 Kubernetes add the ability to orchestrate containers across
 multi-host installations.
 
+[discrete]
+[id="nodes-containers-memory"]
+== About containers and RHEL kernel memory
+
+Due to RHEL behavior, a container on a node with high CPU usage might appear to consume more memory than expected. The higher memory consumption could be caused by the RHEL kernel kmem_cache. The RHEL kernel creates a kmem_cache for each each cgroup, which consumes kernel memory. For added performance, the kmem_cache contains a cpu_cache and a node cache for any NUMA nodes. 
+
+The amount of memory stored in those caches is proportional to the number of CPUs that the system uses. As a result, a higher number of CPUs results in a greater amount of kernel memory held being in these caches, Higher amounts of kernel memory in these caches can cause {product-title} containers to exceed the configured memory limits, resulting in the container being killed. 
+
+To avoid losing containers due to kernel memory issues, you should ensure that containers request sufficient memory. You can use the following formula to ensure that your containers have enough memory, where `nproc` is the number of process reported by the `nproc` command:
```

Comment:
Maybe instead of `You can use the following formula to ensure that your containers have enough memory` 
`You can use the following formula to estimate the amount of memory will by consumed by the kmem_cache`
I guess I mostly want to imply that the command below should be the lower limit of their container's request, and their container request should be `*$(nproc) * 1/2 MiB* + *estimated requirements of the container*`

Reply:
@haircommander Thanks for the input!